lib/App/Sqitch/Command/plan.pm

```diff
@@ -236,6 +236,7 @@ sub execute {
             change_id     => $change->id,
             change        => $change->name,
             note          => $change->note,
+            file          => $change->file,
```

Comment:
I guess it's not a direct change to add it to the `log` command, too. Bit of a pity, since the `log` and `plan` commands otherwise support exactly the same formats.

Reply:
I looked in it, but decided not to, because on this moment it lacks some of `plan` format options anyway. Like `%r` or `%t` for example (though they are in documentation). They are in `ItemFormatter.pm` (it is the same class for both `plan` and `log`), but values for them do not passed in `log.pm`.